gateway-service/src/main/java/org/zowe/apiml/gateway/caching/CachingServiceClientRest.java

```diff
@@ -56,26 +56,33 @@ public CachingServiceClientRest(
 
     @PostConstruct
     void updateUrl() {
+        // Lazy initialization of GatewayClient's ServerAddress may bring invalid URL during initialization
```

Comment:
does it need to be PostConstruct in that case?

Reply:
Probably not